library-backend/library-app/src/main/java/org/internship/library/app/controller/RedirectController.java

```diff
@@ -0,0 +1,48 @@
+package org.internship.library.app.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.view.RedirectView;
+
+import javax.servlet.http.HttpServletResponse;
+
+@Controller
+@RequestMapping(path="redirect")
+public class RedirectController {
+
+    /**
+     * Redirect to external URL
+     * @param model Used to build URL attributes
+     * @return The external URL
+     */
+    @GetMapping("/redirectPrefix")
```

Comment:
Let's numerotate each of the with /redirect/1, 2 and 3 and in the javadoc add a short explanation on what's happening 'behind the scenes'.

E.g. how does the 302 one work?
how does the other one work?